src/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
-import {ContextualSaveBarProps, useFrame} from '../../utilities/frame';
+import {
+  ContextualSaveBarProps as ContextualSaveBarProps1,
+  useFrame,
+} from '../../utilities/frame';
 
 // The script in the styleguide that generates the Props Explorer data expects
 // a component's props to be found in the Props interface. This silly workaround
 // ensures that the Props Explorer table is generated correctly, instead of
-// crashing if we write `ContextualSaveBar extends React.Component<ContextualSaveBarProps>`
-export interface ContextualSaveBarProps extends ContextualSaveBarProps {}
```

Comment:
TS 3.7 stopped you having a local type with the same name as an import https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#local-and-imported-type-declarations-now-conflict

Reply:
I thumbs upped your comment but I do not thumbs up anything that's going on here